ProfileView/src/index.ts

```diff
@@ -49,8 +49,6 @@ export default class CjaasProfileWidget extends LitElement {
   @property({ type: String, attribute: "tape-read-token" }) tapeReadToken:
     | string
     | null = null;
-  @property({ type: String, attribute: "profile-read-token" })
-  profileReadToken: string | null = null;
```

Comment:
Breaking change, making this a major version update

Reply:
Updated ProfileView to 3.0.0